src/libs/routes/errorHandler.ts

```diff
@@ -0,0 +1,11 @@
+const errorHandler = ((err , req , res , next) => {
+    console.log('Error', err);
```

Comment:
check whether headers are sent or not if not throw error

Reply:
done